dxr/static/css/dxr.css

```diff
@@ -148,6 +148,10 @@ td#line-numbers span {
     text-align: right;
     padding: 0 0.5rem;
 }
+tr.binary_row {
+    font-style: italic;
```

Comment:
I'm leaning toward losing the italics; I think the opacity (which I love) and removing the hover state (which we should do) are hints enough.